static/js/admin.js

```diff
@@ -657,6 +657,15 @@ function _setup_page() {
                         ui.report_success(i18n.t("waiting period threshold changed!"), waiting_period_threshold_status);
                     }
                 }
+                // Check if no changes made
+                var no_changes_made = true;
+                for(var key in response_data) {
+                    if(['msg', 'result', 'authentication_methods'].indexOf(key) < 0)
+                        no_changes_made = false;
+                }
```

Comment:
This looks a lot better!

Are you sure `authentication_methods` should be in this list?  Seems like that may suggest a bug elsewhere; at least, reading the backend code, it should only be set if the authentication methods list is being changed...

Also, can you fix the whitespace in your JS style?